plat/nvidia/tegra/soc/t132/plat_psci_handlers.c

```diff
@@ -132,7 +132,7 @@ int tegra_soc_pwr_domain_off(const psci_power_state_t *target_state)
 
 int tegra_soc_pwr_domain_suspend(const psci_power_state_t *target_state)
 {
-#if DEBUG
+#if ENABLE_ASSERTIONS
```

Comment:
@antonio-nino-diaz-arm 

I wonder why this file was changed. Are we supposed to replace all such #if DEBUG checks with this new macro?